src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (number1, number2, operation) => {
+  const num1 = Big(number1);
+  const num2 = Big(number2);
+  let total = 0;
+
+  const divide = () => (num2 === 0 ? 'NaN' : num1.div(num2));
+
+  const modulus = (num1, num2) => (!num2 ? num1.times(0.01) : num1.times(num2).times(0.01));
```

Comment:
Job well done writing `function modulus` but you don't have to do the calculating yourself, that where the use of `Big` comes in -> [big js library](http://mikemcl.github.io/big.js/#mod)

example: case modifier will be -> `num1.mod.(num2)`